src/vim9expr.c

```diff
@@ -811,7 +873,34 @@ compile_load(
 	    }
 	}
 	if (gen_load)
-	    res = generate_LOAD(cctx, ISN_LOAD, idx, NULL, type);
+	{
+	    // Optimization: check if can generate LOAD CLASSMEMBER.
+	    int did_classmember = FALSE;
+	    if (type->tt_type == VAR_OBJECT
+		&& (type->tt_class->class_flags & CLASS_INTERFACE) == 0
+		&& !gen_load_outer)
+	    {
+		// Got an object that is not an interface.
+		int is_class;
+		char_u *name_end;
```

Comment:
The coding convention is to use tabs between the types and the variable names and align the variable names.

Reply:
Right, I've just been getting my head wrapped around that. I've actually started doing that, some some cruft left around.

Reply:
> The coding convention is to use tabs between the types and the variable names and align the variable names.

Do you mean "hard tabs"? The modeline has `sts=4`. I don't think I know how to enter a hard tab when `sts` is set.

Reply:
I meant aligning the variable names using a soft-tab.